roles/setup/handlers/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Restart Receptor
+  ansible.builtin.systemd:
```

Comment:
Same comment about service instead of systemd.

Reply:
i think given our service confs are specific to systemd at the moment, we can just use `systemd` module